packages/react/package.json

```diff
@@ -87,7 +87,7 @@
     "@github/markdown-toolbar-element": "^2.1.0",
     "@github/paste-markdown": "^1.4.0",
     "@github/relative-time-element": "^4.1.2",
-    "@lit-labs/react": "^1.1.1",
```

Comment:
Lit's react package has graduated from labs. We should use the latest version. From their [npm package readme](https://www.npmjs.com/package/@lit-labs/react):

> This package has graduated from labs! It is now available as @lit/react. This package is just a proxy that re-exports @lit/react. As a result, while it will no longer be updated, it will continue to work and get updates through the ^1.0.0 version range of @lit/react. This should reduce duplication of code while the ecosystem migrates their imports away from @lit-labs/react.

Reply:
@keithamus just a heads up, we ran into an issue when moving to newer versions of `@lit-labs/react` (I think v2+?) which is what graduated into `@lit/react` from what I can tell. Specifically, it was when the packages moved to ESM only.

I tried to capture the issues over in: https://github.com/primer/react/issues/3710. Mostly they're due to us having to bundle ESM-only deps as CommonJS since we still have to support that in the current major version. When we do that, the way they do conditional exports unfortunately no longer works 😞 

We have plans to move to ESM only in the next major AFAIK but couldn't think of something in the current major that would support this. Would love to know what you think

Reply:
I'll just bump it on the minor then.